ts/masters/highcharts.src.ts

```diff
@@ -29,6 +29,7 @@ import '../Core/Legend.js';
 import Chart from '../Core/Chart/Chart.js';
 import '../Extensions/ScrollablePlotArea.js';
 import '../Extensions/Stacking.js';
+import DataTable from '../Data/DataTable.js';
```

Comment:
Do we need this in the core at this point? This import is probably responsible for the bundle size going up by 3 kB (gzipped).

At this point we are not using the data layer in the core. The end goal is to replace the series' internal data handling with the data layer, so when we have accomplished that, the bundle size should be balanced. But we should not load redundant code into a bundle that will reach production with the next release.